### DISCUSSION TITLE
Possibly outdated documentation

### DISCUSSION THREAD

**Discussion by refaelsh** [polybar/polybar#2809] [Q&A]
Regarding this: https://github.com/polybar/polybar/wiki/User-contributed-modules#other-languages.
I think it is something very old and outdated, since polybar with the default config file has the functionality from the above link.

**Comment by patrick96** [comment]
I don't use xmonad, so it don't really know much about it. What about the script is outdated? Does it not work anymore?

It's fine if polybar has similar functionality built-in, we can still have alternatives on the wiki. Especially since the xworkspaces module only works on WMs that support the EWMH specification and I don't know to what extent xmonad supports that.

**Comment by refaelsh** [comment]
> What about the script is outdated?

It is no longer needed since polybar with the default config has this functionality.

> Does it not work anymore?

No idea. Have not tried it.

> to what extent xmonad supports that

By default, no. But is easily fixable by adding one line to XMonad config.

**Reply by patrick96** [reply]
I think we can leave it in the wiki. It's still a good starting point if people want to hook into xmonad directly.
